### PR TITLE
add suport for pythtest and pythnet

### PR DESCRIPTION
**Pull request by cctdaniel**

### CODE REVIEW

File: observer.py [hunk 1]
```diff
@@ -181,7 +187,7 @@ async def run_coingecko_get_price():
         nonlocal coingecko_prices
         while True:
             coingecko_prices = get_coingecko_prices([x for x in symbol_to_id_mapping])
-            await asyncio.sleep(2)
+            await asyncio.sleep(3)
```
Comment (cctdaniel) [inline observer.py:190]: getting too many requests error from coingecko hence bumping sleep time

Comment (thmzlt) [reply observer.py:190]: Do they announce the API rate limits? There's [a nice Python library I've been using](https://pypi.org/project/throttler/) to handle the rate of requests.

Comment (cctdaniel) [reply observer.py:190]: this seems helpful thanks!

**Review by thmzlt — COMMENTED** [review]
Looks like you ran a code formatter through the code. It would be great if you could isolate that on a separate commit so we could review the significant changes in a shorter diff.

**Review by thmzlt — APPROVED** [review]
Looks good to me.

File: observer.py [hunk 2]
```diff
@@ -116,12 +118,12 @@ async def run_alerts():
                     )
                     asyncio.sleep(0.4)
                     continue
-
+
```
Comment (thmzlt) [inline observer.py]: You might want to add a trim whitespace formatter on your editor.

File: pyth_observer/coingecko.py [hunk 3]
```diff
@@ -3,30 +3,42 @@
...
-api_to_symbol_mapping = {symbol_to_id_mapping[x]['api']: x for x in symbol_to_id_mapping}
+api_to_symbol_mapping = {
+    symbol_to_id_mapping[x]["api"]: x for x in symbol_to_id_mapping
+}
 
 
-def get_coingecko_prices(symbols):
+# CoinGecko free API limit: 10-50 (varies) https://www.coingecko.com/en/api/pricing
+# However prices are updated every 1-10 minutes: https://www.coingecko.com/en/faq
+# Hence we only have to query once every minute.
```
Comment (cctdaniel) [inline pyth_observer/coingecko.py:29]: this is the part where the stale prices comes in @thmzlt

File: observer.py (hunk 1 above)
Comment (wofvh) [reply observer.py]: 형 good !

**Pull request merged**